src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -5342,18 +5382,19 @@ namespace SimAirServingZones {
 							Xs = 0.0;
 						}
 						if ( FinalSysSizing( AirLoopNum ).OAAutoSized && SysSizing( CurOverallSimDay, AirLoopNum ).DesCoolVolFlow > 0 ) {
-							NumZonesCooled = AirToZoneNodeInfo( AirLoopNum ).NumZonesCooled;
+							int NumZonesCooled = AirToZoneNodeInfo( AirLoopNum ).NumZonesCooled;
 							MinCoolingEvz = 1.0;
 							VozSumClgBySys( AirLoopNum ) = 0.0;
-							for ( ZonesCooledNum = 1; ZonesCooledNum <= NumZonesCooled; ++ZonesCooledNum ) {
-								CtrlZoneNum = AirToZoneNodeInfo( AirLoopNum ).CoolCtrlZoneNums( ZonesCooledNum );
+							for ( int ZonesCooledNum = 1; ZonesCooledNum <= NumZonesCooled; ++ZonesCooledNum ) {
+								int CtrlZoneNum = AirToZoneNodeInfo( AirLoopNum ).CoolCtrlZoneNums( ZonesCooledNum );
+								int TermUnitSizingIndex = AirToZoneNodeInfo( AirLoopNum ).TermUnitCoolSizingIndex( ZonesCooledNum );
 
 								// Zone air secondary recirculation fraction
-								Er = FinalZoneSizing( CtrlZoneNum ).ZoneSecondaryRecirculation;
-								Ep = FinalZoneSizing( CtrlZoneNum ).ZonePrimaryAirFraction;
-								ZoneOAFrac = FinalZoneSizing( CtrlZoneNum ).ZpzClgByZone;
-								ZoneEz = FinalZoneSizing( CtrlZoneNum ).ZoneADEffCooling;
-								VozClg = FinalZoneSizing( CtrlZoneNum ).VozClgByZone;
+								Er = TermUnitFinalZoneSizing( TermUnitSizingIndex ).ZoneSecondaryRecirculation;
+								Ep = TermUnitFinalZoneSizing( TermUnitSizingIndex ).ZonePrimaryAirFraction;
+								ZoneOAFrac = TermUnitFinalZoneSizing( TermUnitSizingIndex ).ZpzClgByZone;
+								ZoneEz = TermUnitFinalZoneSizing( TermUnitSizingIndex ).ZoneADEffCooling;
+								VozClg = TermUnitFinalZoneSizing( TermUnitSizingIndex ).VozClgByZone;
```

Comment:
These are also used to calculate a zone value, what if 2 air loops?

    Fa = Ep + ( 1.0 - Ep ) * Er;
    FaByZoneCool( CtrlZoneNum ) = Fa;

Reply:
I know - the whole Std62 calc section doesn't work with more than one airloop/terminal unit in a zone. This needs a big redo but that may have to be yet another pull request.